docs/book/src/developers/development.md

```diff
@@ -151,6 +153,17 @@ development will span both CAPZ and CAPI, then follow the [CAPI and CAPZ instruc
 
 #### Tilt for dev in CAPZ
 
+<aside class="note warning">
+  <h2>Warning</h2>
+  <p>
+    To use the internal load balancer (LB) for internode communication in your workload cluster, follow the
```

Comment:
Let's replace "To use the internal load balancer (LB) for internode communication"

with

"To use an internal load balancer (ILB) intra-cluster node-apiserver traffic"

I'd do the same below as well

Reply:
Updated :)